lib/rubocop/cop/style/conditional_assignment.rb

```diff
@@ -207,7 +207,7 @@ class ConditionalAssignment < Cop
         MAX = 'Max'.freeze
         SINGLE_LINE_CONDITIONS_ONLY = 'SingleLineConditionsOnly'.freeze
         WIDTH = 'Width'.freeze
-        METHODS = [:[]=, :<<, :=~, :!~, :<=>].freeze
+        METHODS = [:+, :-, :[]=, :<<, :=~, :!~, :<=>].freeze
```

Comment:
Not sure whether this is a good idea or not. Perhaps it should be a separate config option. Also - there are other arithmetic operators that can be taken into account.